### DISCUSSION TITLE
Make "how to install a napari plugin" easier to find.

### DISCUSSION THREAD

**Discussion by tlambert03** [chanzuckerberg/napari-hub#375] [Ideas]
#### Description
I was browsing our docs on napari.org, and feeling like [this page](https://napari.org/plugins/stable/find-and-install-plugin.html) (finding and installing a napari plugin) feels out of place in a section that is mostly geared towards people _making_ plugins.  It seemed like the hub would be a better place for that, so I just came to check that it was there.

I first tried clicking the FAQ but didn't see it (that was more geared to devs who might want to edit their listings). I know from experience that each individual plugin page has an "install" button that contains information on how to that plugin, but it might not be immediately obvious to someone that they have to first _pick_ a specific plugin before they get to learn how one generally goes about installing a plugin into napari.

Those in-page links are nice, but it would be great to have an easier-to-discover page that fulfills the promise in the header: "learn how to install into napari"

**Comment by justinelarsen** [comment]
I think this makes sense. The team is currently exploring a new design to the hub homepage which could possibly feature content like this with links to more detailed info. @neuromusic @liaprins-czi Something to keep in mind for the new design and to figure out the best location for it.
